openmdao/drivers/tests/test_pyoptsparse_driver.py

```diff
@@ -1483,9 +1483,13 @@ def test_debug_print_option(self):
         self.assertTrue(len([s for s in output if s.startswith('comp.f_xy')]) > 1,
                         "Should be more than one comp.f_xy printed")
 
+
+@unittest.skipIf(OPT is None or OPTIMIZER is None, "only run if pyoptsparse is installed.")
 class TestPyoptSparseFeature(unittest.TestCase):
 
     def setUp(self):
+        from openmdao.utils.general_utils import set_pyoptsparse_opt
+        OPT, OPTIMIZER = set_pyoptsparse_opt('SNOPT')
         if OPT is None:
             raise unittest.SkipTest("pyoptsparse is not installed")
```

Comment:
I would probably change the skip message to say that SNOPT is not available as an optimizer here?

Reply:
This part of the skip check just looks for pyoptsparse. The second part is what looks for SNOPT.  But, I did take the SLSQP mention out.

Reply:
OH, I guess I was confused by the skiptest 5 lines up that I thought was only running this test if pyoptsparse was installed.  Then i thought setting the OPT to SNOPT was checking to see further if SNOPT were installed?